### DISCUSSION TITLE
Open source OpenDTU hardware

### DISCUSSION THREAD

**Discussion by chungpc** [tbnobody/OpenDTU#2239] [General]
Hi all,

I'm currently developing a open source hardware for the OpenDTU. This will allow NRF and CMT to be exist at the same board while having ATM90E26 as the energy monitoring to allow a net zero export feature. Is anyone here interested with that? Device is in stackable format.

![image](https://github.com/user-attachments/assets/ae29b50c-0882-4e41-b283-e6fc2617f793)
![image](https://github.com/user-attachments/assets/a0408c1b-efb2-477a-bfd5-0735db63a9e6)
![image](https://github.com/user-attachments/assets/cfb4e3ff-ab15-47af-acde-d79b2e674d2f)
![image](https://github.com/user-attachments/assets/b652aa43-9fe9-48e2-8244-4f7a7a337d3a)
![image](https://github.com/user-attachments/assets/e61b0c19-edf7-45af-9915-2932cab29e26)

**Comment by stefan123t** [comment]
Inetersting setup with a stack of PCBs.

As you want to add zero export functionality you may want to look at the Dynamic Power Limiter function and Power Meter options in the daughter project[ OpenDTU-OnBattery](https://github.com/helgeerbe/OpenDTU-OnBattery/) ?

I have described a [possible PM+DPL setup](https://github.com/tbnobody/OpenDTU/issues/272#issuecomment-2219644352) with a Shelly Pro3EM as **Power Meter (PM)** here in issue #272.
